vizro-core/tests/unit/vizro/models/_controls/test_filter.py

```diff
@@ -244,6 +432,77 @@ def test_check_target_present_invalid(self):
             Filter(column="foo", targets=["invalid_target"])
 
 
+@pytest.mark.usefixtures("managers_column_only_exists_in_some")
+class TestFilterCall:
```

Comment:
There are three resolutions for these tests in my opinion:
1. Improve/refine the implementation a bit (if there are any PR comments).
2. Remove these tests because they are not really relevant.
3. Merge these tests with the `_get_min_max` and `_get_options` tests.

Reply:
I think it's good how you have it 👍 If they cause problems in future we can remove them, but I'd rather have too many tests rather than too few.

The only comment I have is instead specifying filter properties manually like  this:
```
filter = vm.Filter(
            column="column_categorical", targets=["column_categorical_exists_1", "column_categorical_exists_2"]
        )
        filter._column_type = "categorical"
        filter.selector = vm.Dropdown(id="test_selector_id")
```
why can't we do this?
```
filter = vm.Filter(
            column="column_categorical", targets=["column_categorical_exists_1", "column_categorical_exists_2"],
 selector=...)
filter.pre_build() # if needed
```

Comment applies in many places.

Reply:
This makes sense so I aligned tests.